tests/integration/components/polaris-card-test.js

```diff
@@ -423,4 +423,64 @@ module('Integration | Component | polaris card', function(hooks) {
       'renders the correct custom section header content'
     );
   });
+
+  test('can handle primary & secondary footer actions', async function(assert) {
+    this.setProperties({
+      saveAction: {
+        text: 'Save',
+        disabled: true,
+        onAction() {
+          assert.step('primary-action');
+        },
+      },
+      cancelAction: {
+        text: 'Cancel',
+        onAction() {
+          assert.step('secondary-action');
+        },
+      },
+    });
+
+    await render(
+      hbs`{{polaris-card primaryFooterAction=saveAction secondaryFooterAction=cancelAction}}`
+    );
+
+    let footerClass = '.Polaris-Card__Footer';
+    let footerButtons = `${footerClass} button`;
+    let primaryBtn = '[data-test-id="primaryFooterAction"]';
+    let secondaryBtn = '[data-test-id="secondaryFooterAction"]';
+
+    assert.dom(footerClass).exists();
+    assert.dom(footerButtons).exists({ count: 2 });
+
+    await click(secondaryBtn);
+    assert.verifySteps(['secondary-action']);
+    assert.dom(primaryBtn).isDisabled();
+
+    this.set('saveAction.disabled', false);
+    await click(primaryBtn);
+    assert.verifySteps(['primary-action']);
+    assert.dom(primaryBtn).isNotDisabled();
+
+    assert.dom(primaryBtn).hasText('Save');
+    assert.dom(secondaryBtn).hasText('Cancel');
+
+    await render(hbs`{{polaris-card primaryFooterAction=saveAction}}`);
+
+    assert.dom(footerButtons).exists({ count: 1 });
+    assert.dom(primaryBtn).exists();
+    assert.dom(secondaryBtn).doesNotExist();
+
+    this.set('saveAction.loading', true);
+    assert.dom(`${primaryBtn} .Polaris-Button__Spinner`).exists();
+
+    await render(hbs`{{polaris-card secondaryFooterAction=cancelAction}}`);
+
+    assert.dom(footerButtons).exists({ count: 1 });
+    assert.dom(secondaryBtn).exists();
+    assert.dom(primaryBtn).doesNotExist();
+
+    this.set('cancelAction.loading', true);
+    assert.dom(`${secondaryBtn} .Polaris-Button__Spinner`).exists();
```

Comment:
`loading` buttons have the `Polaris-Button--loading` class, so maybe it makes more sense to look for that here rather than a spinner (or check both 😛)?